### PR TITLE
Add Persian Language

### PR DESCRIPTION
**Pull request by omidmaz1378**
Hi , Please add Persian Language

### CODE REVIEW

**Comment by amirhosseinhpv** [comment]
I've tried #216 and here is the latest version with Persian and multi-lingual language selection: https://github.com/amirhosseinhpv/WhyNotWin11

File: Langs/1065.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+DirectX 12, but no WDDM2=دایرکت ایکس 12 است اما WDDM2 نیست
+No DirectX 12 or WDDM2=دایرکت ایکس 12 و WDDM2 نیست
+Test Build?=تست نسخه؟
+You're running a newer build than publically Available!=شما نسخه جدیدتری از عموم را استفاده میکنید
+Up to Date=به روز
+You're running the latest build!=آخرین ورژن را استفاده میکنید
+Unable to Check for Updates=امکان چک کردن آپدیت نیست
+Unable to load release data.=اطلاعات ریلیز را لود نکرد
+Invalid Data Received!=دیتا با خطا دریافت شد
+Invalid Release Tags Received!=تگ های ریلیز با خطا دریافت شد
```
Comment (micwoj92) [inline Langs/1065.lang:56]: Hello, I quickly check it with google translate and it seems like you did phonetic "release"? Could you explain?
image

Comment (omidmaz1378) [reply Langs/1065.lang:56]: hi, this is an English word which there isn't any good translation in Persian, so we translate it phonetically.

Comment (omidmaz1378) [reply Langs/1065.lang:56]: absolutely

Comment (amirhosseinhpv) [reply Langs/1065.lang:56]: You can check Persian translation in my fork, I have MA in English studies and worked as TEFL teacher for 5 years. Translations in my fork are free to use, and those are actual words we use not robotic translations. Thanks

**Pull request merged**